jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -75,9 +75,9 @@ def parse_data(app, file_obj, data_type=None, data_label=None):
     #  into something glue can understand.
     elif isinstance(file_obj, Spectrum1D):
         if file_obj.flux.ndim == 3:
-            _parse_spectrum1d_3d(app, file_obj)
+            _parse_spectrum1d_3d(app, file_obj, data_label=data_label)
```

Comment:
It is highly annoying to me that `data_label` is ignored when you pass `Spectrum1D` into Cubeviz, so I fixed it here. Unless there is a good reason not to do this? I cannot think of any good reason.